javav2/example_code/appsync/src/main/java/com/example/appsync/CreateDataSource.java

```diff
@@ -0,0 +1,93 @@
+// snippet-sourcedescription:[CreateDataSource.java demonstrates how to create an AWS AppSync data source that uses Amazon DynamoDB.]
+//snippet-keyword:[AWS SDK for Java v2]
+// snippet-service:[AWS AppSync]
+// snippet-keyword:[Code Sample]
+// snippet-sourcetype:[full-example]
+// snippet-sourcedate:[04-01-2022]
+// snippet-sourceauthor:[scmacdon - AWS]
+/*
+   Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+   SPDX-License-Identifier: Apache-2.0
+*/
+
+package com.example.appsync;
+
+//snippet-start:[appsync.java2.create_ds.import]
+import software.amazon.awssdk.regions.Region;
+import software.amazon.awssdk.services.appsync.AppSyncClient;
+import software.amazon.awssdk.services.appsync.model.DynamodbDataSourceConfig;
+import software.amazon.awssdk.services.appsync.model.CreateDataSourceRequest;
+import software.amazon.awssdk.services.appsync.model.DataSourceType;
+import software.amazon.awssdk.services.appsync.model.CreateDataSourceResponse;
+import software.amazon.awssdk.services.appsync.model.AppSyncException;
+//snippet-end:[appsync.java2.create_ds.import]
+
+/**
+ * Before running this Java V2 code example, set up your development environment, including your credentials.
+ *
+ * For more information, see the following documentation topic:
+ *
+ * https://docs.aws.amazon.com/sdk-for-java/latest/developer-guide/get-started.html
+ */
+public class CreateDataSource {
+
+    public static void main(String[] args) {
+
+        final String USAGE = "\n" +
+                "Usage: " +
+                "   <apiId> <name> <dsRole> <tableName>\n\n" +
+                "Where:\n" +
+                "   apiId - The id of the API (you can obtain the value from the AWS Management console). \n\n" +

```

Comment:
(you can obtain the value from the AWS Management console) --> (you can get this value from the AWS Management Console)

Reply:
fixed